augraphy/augmentations/voronoi.py

```diff
@@ -150,11 +150,15 @@ def apply_augmentation(self):
             (nx, ny, ng),
             (perlin_x, perlin_y),
         )
-        image = Image.fromarray(img_array)
-        image.save("images/Voronoi_example.png", "PNG", dpi=(300, 300))
-        # reads it in a format so that it can be applied as a background pattern to the original image
-        mesh = cv2.imread("images/Voronoi_example.png")
-        os.remove("images/Voronoi_example.png")
+        # try if it is able to save and read image properly, might facing permission issue
+        try:
+            image = Image.fromarray(img_array)
+            image.save(os.getcwd() + "/Voronoi_example.png", "PNG", dpi=(300, 300))
+            # reads it in a format so that it can be applied as a background pattern to the original image
+            mesh = cv2.imread(os.getcwd() + "/Voronoi_example.png")
+            os.remove(os.getcwd() + "/Voronoi_example.png")
+        except Exception:
+            mesh = img_array
```

Comment:
@ss756  So there is a problem with the old code here because the folder `images` might not exist in the directory. I removed that directory and change it into a try-except in case there's any permission issue while writing the file. If there is any error, it will just use the `img_array` instead.